src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -123,6 +123,8 @@ func forceEOF(yylex interface{}) {
 	optimizeOptionEnum    OptimizeOptionEnum
 	checkOptionEnum       CheckOptionEnum
 	checkOptionList       CheckOptionList
+	delete_options_list   DeleteOptionList
+	delete_option         DeleteOptionEnum 
```

Comment:
Maintain the same format as before：delete_option ->?

Reply:
fixed.